grid/style.css

```diff
@@ -0,0 +1,73 @@
+@import url("../global.css");
+
+.grid-container {
+  width: 100%;
+}
+
+.grid-row {
+  margin-bottom: 10px;
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.grid-col {
+  flex-basis: 0;
+  flex-grow: 1;
+  max-width: 100%;
+  padding: 0.5rem;
+}
+
+.grid-col-30 {
+  flex: 0 0 30%;
+  max-width: 30%;
+}
+.grid-col-40 {
+  flex: 0 0 40%;
+  max-width: 40%;
+}
+.grid-col-50 {
+  flex: 0 0 50%;
+  max-width: 50%;
+}
+.grid-col-60 {
+  flex: 0 0 60%;
+  max-width: 60%;
+}
+.grid-col-70 {
+  flex: 0 0 70%;
+  max-width: 70%;
+}
+.grid-col-80 {
+  flex: 0 0 80%;
+  max-width: 80%;
+}
+.align-items-start {
+  align-items: flex-start !important;
+}
```

Comment:
Suggestion- You should avoid using important unless it's not required as it will override all the other stylings.
Rest everything seems good.